### PR TITLE
Updating static content for metazoa 108(55)

### PR DESCRIPTION
**Pull request by vsitnik**
Metazoan 108 static content updated

### CODE REVIEW

**Review by ens-ap5 — APPROVED** [review]
I'm not going to read every file, but I checked one of them and the Markdown looks fine. They can be corrected if anything looks amiss on staging.

File: metazoa/species/Anneissia/Anneissia_japonica/Anneissia_japonica_gca011630105v1_annotation.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 ----------
 
 The annotation presented is derived from annotation submitted to
-[INSDC](http://www.insdc.org) with the assembly accession [GCA_011630105.1](http://www.ebi.ac.uk/ena/data/view/GCA_011630105.1).
+[INSDC](http://www.insdc.org) with the assembly accession [GCA\_011630105.1](http://www.ebi.ac.uk/ena/data/view/GCA_011630105.1).
```
Comment (ens-LCampbell) [inline metazoa/species/Anneissia/Anneissia_japonica/Anneissia_japonica_gca011630105v1_annotation.md:5]: Im not entirely sure adding the backslash makes any difference to display or linking to URLs. It doesn't hurt to have them but I don't think they are necessitated for future ref

File: metazoa/species/Daphnia/Daphnia_magna/Daphnia_magna_gca020631705v2_about.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+**About *Daphnia magna***
+-------------------------
+*Daphnia magna* is a small planktonic crustacean that belongs to the 
+subclass Phyllopoda.
```
Comment (ens-LCampbell) [inline metazoa/species/Daphnia/Daphnia_magna/Daphnia_magna_gca020631705v2_about.md:4]: I would be tempted to expand a bit on this organism in terms of description. Perhaps could be useful to add in...

"It inhabits a variety of freshwater environments and is broadly distributed throughout the northern hemisphere, and in South Africa. 

D. magna is used in many different fields of research, such as ecotoxicology, population genetics, the evolution of sex, phenotypic plasticity, ecophysiology (including global change biology) and host-parasite interactions."

File: metazoa/species/Homarus/Homarus_americanus/Homarus_americanus_gca018991925v1_about.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+**About *Homarus americanus***
+-------------------------
+The American lobster (*Homarus americanus*) is a species of lobster
+found on the Atlantic coast of North America, chiefly from Labrador to
+New Jersey. It is also known as Atlantic lobster, Canadian lobster,
+true lobster, northern lobster, Canadian Reds, or Maine lobster. It
+can reach a body length of 64�cm (25�in), and a mass of over 20
```
Comment (ens-LCampbell) [inline metazoa/species/Homarus/Homarus_americanus/Homarus_americanus_gca018991925v1_about.md:7]: Encoded chars not displaying correctly in md "?".

File: metazoa/species/Homarus/Homarus_americanus/Homarus_americanus_gca018991925v1_about.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+kilograms (44�lb), making it not only the heaviest crustacean in the
+world, but also the heaviest of all living arthropod species. Its
+closest relative is the European lobster Homarus gammarus, which can
```
Comment (ens-LCampbell) [inline metazoa/species/Homarus/Homarus_americanus/Homarus_americanus_gca018991925v1_about.md:10]: Would you mind changing this sentence from "Its
closest relative is the European lobster Homarus gammarus," to "The
closest relative to _H. americanus_ and the only other species within the genus is the European lobster _Homarus gammarus_."

File: metazoa/species/Anneissia/Anneissia_japonica/Anneissia_japonica_gca011630105v1_annotation.md (hunk 1 above)
Comment (vsitnik) [reply metazoa/species/Anneissia/Anneissia_japonica/Anneissia_japonica_gca011630105v1_annotation.md:5]: It's better to escape special symbols. We had issues with unescaped _ in accession previously.

File: metazoa/species/Daphnia/Daphnia_magna/Daphnia_magna_gca020631705v2_about.md (hunk 2 above)
Comment (vsitnik) [reply metazoa/species/Daphnia/Daphnia_magna/Daphnia_magna_gca020631705v2_about.md:4]: thanks. done.

File: metazoa/species/Homarus/Homarus_americanus/Homarus_americanus_gca018991925v1_about.md (hunk 3 above)
Comment (vsitnik) [reply metazoa/species/Homarus/Homarus_americanus/Homarus_americanus_gca018991925v1_about.md]: cool. thanks! fixed

File: metazoa/species/Daphnia/Daphnia_magna/Daphnia_magna_about.md [hunk 5]
```diff
@@ -6,6 +6,8 @@ subclass Phyllopoda. It inhabits a variety of freshwater environments
 and is broadly distributed throughout the northern hemisphere, and in
 South Africa.
 
+D. magna is used in many different fields of research, such as ecotoxicology, population genetics, the evolution of sex, phenotypic plasticity, ecophysiology (including global change biology) and host-parasite interactions.
```
Comment (ens-LCampbell) [inline metazoa/species/Daphnia/Daphnia_magna/Daphnia_magna_about.md:9]: Cheers Vasily. Just to add a bit more as its an important model organism :)

File: metazoa/species/Homarus/Homarus_americanus/Homarus_americanus_gca018991925v1_about.md (hunk 4 above)
Comment (vsitnik) [reply metazoa/species/Homarus/Homarus_americanus/Homarus_americanus_gca018991925v1_about.md]: updated. thanks!

File: metazoa/species/Daphnia/Daphnia_magna/Daphnia_magna_about.md (hunk 5 above)
Comment (vsitnik) [reply metazoa/species/Daphnia/Daphnia_magna/Daphnia_magna_about.md:9]: updated with the "gca" one

**Review by ens-LCampbell — APPROVED** [review]
I had a scan over all files and only spotted those few issues I made in single line comments. Other than that, I think its fine and happy to accept the PR in its current state.

**Pull request merged**